### ISSUE TITLE
Build docker image multiarch (armv7, arm64)

### ISSUE THREAD

**Issue by ohthehugemanatee** [benbjohnson/litestream#310]
Love the project, it has made it possible to run a  lot of services on my k3s cluster! Thank you. 

Unfortunately the [docker image](https://hub.docker.com/r/litestream/litestream/tags) is only built for amd64, so it will only run on certain nodes.

If you haven't done it before, building container images cross platform is very easy. I would submit a PR if I could find the pipeline where you're doing it!  In a GitHub Actions workflow these are the relevant steps:

``` yaml
      - name: setup qemu
        uses: docker/setup-qemu-action@v1
      - name: setup buildx
        id: buildx
        uses: docker/setup-buildx-action@v1
      - name: Log in to the Container registry
        run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
      - name: build the image and push
        run: |
          cd cluster/Dockerfiles/sidecar-sync && 
          docker buildx build \
          --push \
          --tag ghcr.io/${{ github.actor }}/sidecar-sync:multiarch \
          --platform linux/amd64,linux/arm64,linux/arm/v7 . 
```

You can see a complete example in [one of my own repos](https://github.com/ohthehugemanatee/homenet-documentation/blob/master/.github/workflows/build-sidecar-sync.yaml).

**Comment by benbjohnson** [comment]
I'm glad it's working well for you in k3s. Thanks for the instructions as well. I need to move my Docker builds & my macOS builds to GitHub Actions. I have a couple things to finish up for read replication and then I'll get the Docker images built for the various platforms after that.

**Comment by ohthehugemanatee** [comment]
If you can point me to the source for your current docker builds, I'd be happy to PR it for your review. MacOS builds I'm less confident about.

**Comment by benbjohnson** [comment]
@ohthehugemanatee The Dockerfile lives here in the root of the project. It's pretty barebones: https://github.com/benbjohnson/litestream/blob/main/Dockerfile

As for the release, I usually run the following:

```sh
$ docker build \
--build-arg LITESTREAM_VERSION=v0.3.7 \
-t litestream/litestream:0.3.7 \
-t litestream/litestream:0.3 \
-t litestream/litestream:0 \
-t litestream/litestream:latest .

docker push -a litestream/litestream
```

I'm not a Docker expert so maybe that's a terrible way to do it. :)

**Comment by benbjohnson** [comment]
Closed via #318
